ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -89,9 +89,13 @@ objects:
         #!/bin/bash
         set -euo pipefail
 
+        export PATH=/usr/libexec/origin:$PATH
+
         trap 'touch /tmp/shared/exit' EXIT
         trap 'kill $(jobs -p); exit 0' TERM
 
+        cp "$(which oc)" /tmp/shared/
```

Comment:
POSIX police: we had been using `which` before, but `command -v oc` is [in POSIX][1] while `which` is not.  Obviously both will work for our test image, and we're unlikely to drop `which` but thought I'd point it out just in case ;).

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html